catroid/src/main/java/org/catrobat/catroid/content/bricks/LegoEv3PlayToneBrick.java

```diff
@@ -110,6 +117,26 @@ public View getView(Context context, int brickId, BaseAdapter baseAdapter) {
 				.setTextFieldId(R.id.brick_ev3_tone_duration_edit_text);
 		getFormulaWithBrickField(BrickField.LEGO_EV3_DURATION_IN_SECONDS).refreshTextField(view);
 
+		TextView times = (TextView) view.findViewById(R.id.brick_ev3_tone_seconds);
+
+		if (getFormulaWithBrickField(BrickField.LEGO_EV3_DURATION_IN_SECONDS).isSingleNumberFormula()) {
+			try {
+				times.setText(view.getResources().getQuantityString(
+						R.plurals.second_plural,
+						Utils.convertDoubleToPluralInteger(getFormulaWithBrickField(BrickField.LEGO_EV3_DURATION_IN_SECONDS)
+								.interpretDouble(ProjectManager.getInstance().getCurrentSprite()))
+				));
+			} catch (InterpretationException interpretationException) {
+				Log.d(getClass().getSimpleName(), "Couldn't interpret Formula.", interpretationException);
+			}
+		} else {
+
+			// Random Number to get into the "other" keyword for values like 0.99 or 2.001 seconds or degrees
```

Comment:
this will also happen if theres a formula containing a sensor value, etc that is not going to be calculated here, so stays visible as formula, right?
please remove comment, its a bit misleading...

Reply:
done